### PR TITLE
feat(maintner): get SLO rules from leif

### PR DESCRIPTION
**Pull request by BrennaEpp**
Add API call from maintner to leif to get SLOs for the repository every 10 min

### CODE REVIEW

File: drghs-worker/maintnerd/main.go [hunk 1]
```diff
@@ -228,10 +229,64 @@ func main() {
...
 
+func getSlos(ctx context.Context, parent string) ([]*drghs_v1.SLO, error) {
+	dialCtx, cancel := context.WithTimeout(ctx, time.Second)
+	defer cancel()
+
+	conn, err := grpc.DialContext(dialCtx, *sloAddress, grpc.WithInsecure(), grpc.WithBlock())
+	if err != nil {
+		return nil, fmt.Errorf("Error connecting to SLO server: %v", err)
+	}
+	defer conn.Close()
```
Comment (orthros) [inline drghs-worker/maintnerd/main.go:264]: nit: consider looking at maintner-rtr or samplr-rtr to use the grpc-interceptors for automatic retry

**Comment by orthros** [comment]
Just realized I reviewed a draft pr! Sorry 😅

File: drghs-worker/maintnerd/main.go (hunk 1 above)
Comment (BrennaEpp) [reply drghs-worker/maintnerd/main.go:264]: Do we want to add tracing with `grpctrace` as well?

Comment (orthros) [reply drghs-worker/maintnerd/main.go:264]: That would be great but not necessary.

File: drghs-worker/maintnerd/main.go [hunk 2]
```diff
@@ -228,10 +231,73 @@ func main() {
...
 
+func getSlos(ctx context.Context, parent string) ([]*drghs_v1.SLO, error) {
+
+	conn, err := grpc.Dial(
+		*sloAddress,
+		grpc.WithInsecure(),
+		grpc.WithBlock(),
+		grpc.WithUnaryInterceptor(
+			grpc_middleware.ChainUnaryClient(
+				grpctrace.UnaryClientInterceptor(global.Tracer("maintner-leif")),
```
Comment (orthros) [inline drghs-worker/maintnerd/main.go:266]: nit: should just be "leif"

**Pull request merged**